appConfigurations.js

```diff
@@ -288,10 +288,18 @@ const configureAppRoutes = app => {
   });
 
   app.get('/', (req, res) => {
-    res.redirect('/entry');
+    if (Object.hasOwn(req.query || {}, 'forceIba')) {
```

Comment:
wouldn't `req?.query['forceIba']`  do the job?

Reply:
not quite, as if query was undefined then it would throw and error on reading forceIba on undefined, also linter throws a 'Parsing error: Unexpected token .' on null checks like this